PolyEngine/Engine/Src/Imgui/ImguiResource.cpp

```diff
@@ -0,0 +1,82 @@
+#include <EnginePCH.hpp>
+
+#include <Imgui/ImguiResource.hpp>
+#include <Input/KeyBindings.hpp>
+
+using namespace Poly;
+
+ImguiResource::ImguiResource()
```

Comment:
Why make it resource? I think this can safely be system field (use class systems) as it does not hold any state, at least relevant to game.

Reply:
Done